src/modules/dream/Counters.h

```diff
@@ -21,19 +21,21 @@ struct Counters {
   int64_t ErrorESSHeaders;
 
   // DREAM DataParser
-  int64_t Readouts;
-  int64_t DataHeaders;
-  int64_t ErrorDataHeaders;
-  int64_t RingErrors;
-  int64_t FENErrors;
-  int64_t DataLenErrors;
-  int64_t ConfigErrors;
+  int64_t DataHeaders{0};
+  int64_t Readouts{0};
+  int64_t BufferErrors{0};
+  int64_t DataLenErrors{0};
+  int64_t FiberErrors{0};
+  int64_t FENErrors{0};
+
+  int64_t RingMappingErrors{0};
+  int64_t FENMappingErrors{0};
+  int64_t ConfigErrors{0};
 
   //
   int64_t ProcessingIdle;
   int64_t Events;
-  int64_t EventsUdder;
-  int64_t MappingErrors;
+  //int64_t MappingErrors;
```

Comment:
Do we still need MappingErrors? I'd suggest removing this line

Reply:
good catch